*** porn-get	Sat Feb  2 00:43:12 2002
--- porn-get.new	Sat Feb  2 15:44:39 2002
***************
*** 9,17 ****
  version="0.4.0"
  what=`basename $0 |sed s/-.*//`
  conf="${what}.conf"
! dir="./"
  #dir="/usr/share/${what}-get/"
  
  for a in lynx wget; do
      which $a >/dev/null
      if [ ! $? = 0 ]; then echo "$a not found"; exit 1; fi
--- 9,27 ----
  version="0.4.0"
  what=`basename $0 |sed s/-.*//`
  conf="${what}.conf"
! whoami=`whoami`
! photodir="/home/users/$whoami/.${what}-get/photos/"
! dir="/home/users/$whoami/.${what}-get/"
  #dir="/usr/share/${what}-get/"
  
+ if [ ! -d ~/.${what}-get ] ; then
+     mkdir -p ~/.${what}-get
+ fi
+ 
+ if [ ! -d ~/.${what}-get/photos ] ; then
+     mkdir -p ~/.${what}-get/photos
+ fi
+ 
  for a in lynx wget; do
      which $a >/dev/null
      if [ ! $? = 0 ]; then echo "$a not found"; exit 1; fi
***************
*** 43,51 ****
  	cat ${dir}${conf} |grep -v "^#" |grep "^$2 " |while read pkg url fileext; do
  
  	echo "please wait..."
! 	[ ! -e ${dir}${pkg} ] && mkdir -p ${dir}${pkg}
! 	echo "entering directory... ${dir}${pkg}"
! 	cd ${dir}${pkg}
  	referer=`echo $url |cut -d / -f1,2,3`
  	lynx -dump "$url" |grep -E "tp://|tps://" |grep -i "$fileext" |while read num file; do
  	    if [ ! -e `basename $file` ]; then
--- 53,61 ----
  	cat ${dir}${conf} |grep -v "^#" |grep "^$2 " |while read pkg url fileext; do
  
  	echo "please wait..."
! 	[ ! -e ${photodir}${pkg} ] && mkdir -p ${photodir}${pkg}
! 	echo "entering directory... ${photodir}${pkg}"
! 	cd ${photodir}${pkg}
  	referer=`echo $url |cut -d / -f1,2,3`
  	lynx -dump "$url" |grep -E "tp://|tps://" |grep -i "$fileext" |while read num file; do
  	    if [ ! -e `basename $file` ]; then
***************
*** 57,63 ****
  	    fi
  	done
  	wait
! 	find $dir -name $2 -empty -exec rm -r -- '{}' \;
  	cd -
  	done
      ;;
--- 67,73 ----
  	    fi
  	done
  	wait
! 	find $photodir -name $2 -empty -exec rm -r -- '{}' \;
  	cd -
  	done
      ;;
***************
*** 84,97 ****
  	    echo
  	    exit 1;
  	fi
! 	if [ ! -d ${dir}$2 ]; then
  	    echo "E: Couldn't find package $2"
  	    echo
  	    exit 1
  	fi
  	echo "The following packages will be REMOVED:"
  	echo "  $2"
! 	freed=`du -k ${dir}$2/ |awk '{print $1}'`
  	echo "Need to get 0B of archives. After unpacking ${freed}kb will be freed."
  	echo -ne "Do you want to continue? [Y/n] "
  	read reply
--- 94,107 ----
  	    echo
  	    exit 1;
  	fi
! 	if [ ! -d ${photodir}$2 ]; then
  	    echo "E: Couldn't find package $2"
  	    echo
  	    exit 1
  	fi
  	echo "The following packages will be REMOVED:"
  	echo "  $2"
! 	freed=`du -k ${photodir}$2/ |awk '{print $1}'`
  	echo "Need to get 0B of archives. After unpacking ${freed}kb will be freed."
  	echo -ne "Do you want to continue? [Y/n] "
  	read reply
***************
*** 102,108 ****
  	    ;;
  	    *)
  		echo "Removing $2 ..."
! 		rm -rf ${dir}$2
  	    ;;
  	esac
      ;;
--- 112,118 ----
  	    ;;
  	    *)
  		echo "Removing $2 ..."
! 		rm -rf ${photodir}$2
  	    ;;
  	esac
      ;;
